<?xml version="1.0"?>
<launch>
  <node name="final_pipeline" pkg="reconfros_software" type="final_pipeline" output="screen"> </node>
  <node name= "navigation" pkg="reconfros_software" type="navigation" output="screen"> </node> 
  <node name= "rqt_reconfigure" pkg="rqt_reconfigure" type="rqt_reconfigure" output="screen"> </node>
</launch>